doc/man3/RSA_public_encrypt.pod

```diff
@@ -46,17 +48,21 @@ Encrypting user data directly with RSA is insecure.
 
 =back
 
-B<flen> must be less than RSA_size(B<rsa>) - 11 for the PKCS #1 v1.5
-based padding modes, less than RSA_size(B<rsa>) - 41 for
+B<flen> must not be more than RSA_size(B<rsa>) - 11 for the PKCS #1 v1.5
+based padding modes, not more than RSA_size(B<rsa>) - 42 for
 RSA_PKCS1_OAEP_PADDING and exactly RSA_size(B<rsa>) for RSA_NO_PADDING.
```

Comment:
I think we should document that to must contain exactly RSA_size(rsa) bytes.

Reply:
never mind, actually it is written in the paragraph above, but I will double check

Reply:
Instead I added a remark on the flen in RSA_private_decrypt, that it has to be <= RSA_size(rsa)
if leading zero bytes are in the ciphertext, but doing that is not recommended.